### ISSUE TITLE
Autopilot integration does not add contacts to lists

### ISSUE THREAD

**Issue by jhealey5** [verbb/formie#258]
The Autopilot integration in formie successfully adds contacts but it doesn't add the contacts to the selected list.

I've just been looking at the code and the api and seem to have fixed it by moving the `_autopilot_list` property into the `contact` array rather than outside of it.

The lines here: https://github.com/verbb/formie/blob/craft-3/src/integrations/emailmarketing/Autopilot.php#L151-L163

being changed to:

```
'contact' => [
    'Email' => $email,
    'FirstName' => $firstName,
    'LastName' => $lastName,
    'Company' => $company,
    'Phone' => $phone,
    'MobilePhone' => $mobilePhone,
    'Website' => $website,
    'LeadSource' => $leadSource,
    'Status' => $status,
    '_autopilot_list' => $this->listId,
    'custom' => $fieldValues,
]
```

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.4](https://github.com/verbb/formie/releases/tag/1.3.4)
